server/controllers/sitterControl.ts

```diff
@@ -1,21 +1,17 @@
 import express from "express";
 import Sitter from "../models/sitter";
 
-const find = () => {
-  throw new Error("Function not implemented.");
-};
-
 export const getSitters = async (
   req: express.Request,
   res: express.Response
 ) => {
   try {
-    const sitters = await find();
+    const sitters = await Sitter.find();
     res.json(sitters);
-    res.status(201);
+    res.status(200);
   } catch (e) {
-    console.error(e);
-    res.status(400);
```

Comment:
Not success response codes for this cases should be 500(un-handle server error, not client related). I updated it. Same for code below.